docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -7,7 +7,7 @@ Miden assembly provides a set of instructions for performing common cryptographi
 | Instruction    | Stack_input     | Stack_output   | Notes                                      |
 | -------------- | --------------- | -------------- | ------------------------------------------ |
 | hash <br> - *(20 cycles)*  | [A, ...] | [B, ...] | $\{B\} \leftarrow hash(A)$ <BR> where, $hash()$ computes a 1-to-1 Rescue Prime Optimized hash. |
-| hperm  <br> - *(1 cycle)*      | [C, B, A, ...]  | [F, E, D, ...] | $\{D, E, F\} \leftarrow permute(A, B, C)$ <br> where, $permute()$ computes a Rescue Prime Optimized permutation. |
+| hperm  <br> - *(1 cycle)*      | [A, B, C ...]  | [A', B', C', ...] | $\{A', B', C'\} \leftarrow permute(C, B, A)$ <br> Performs a Rescue Prime Optimized permutation to the top 12 elements of the operand stack, where the first 8 elements are the rate and the last 4 the capacity. |
```

Comment:
If I'm not mistaken, it's the opposite: the first 4 elements are the capacity, and the remainder 8 elements are the rate. We should probably mention too that the expected output is the 4 elements starting at the fourth element (i.e. `4..8`)

Reply:
To clarify:

- `0..4` - capacity
- `4..12` - rate